images/airflow/2.9.2/BillOfMaterials/2.9.2-dev/Python-Packages-BOM.txt

```diff
@@ -5611,10 +5611,10 @@ Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 /usr/local/airflow/.local/lib/python3.11/site-packages/boto3-1.34.106.dist-info/NOTICE
 
 boto3-stubs
-1.35.6
```

Comment:
In my change, the version is getting downgraded. I believe I am missing a step which ensures that the latest safe version is picked.

Reply:
This will be resolved in the next commit.